docs/package.json

```diff
@@ -25,6 +25,11 @@
       "last 1 safari version"
     ]
   },
+  "resolutions": {
+    "@babel/helpers": "7.27.0",
+    "@babel/runtime": "^7.26.10",
+    "@babel/runtime-corejs3": "^7.26.10"
```

Comment:
Should we use a fixed version here to keep them in sync? I know babel is picky with that (and might be part of the reason for the problems in the other PR)

Reply:
I think so yeah, it should match the root. I just pushed it